autometa/common/external/hmmer.py

```diff
@@ -126,7 +130,10 @@ def hmmscan(orfs, hmmdb, outfpath, cpus=0, force=False, parallel=True):
     force : bool, optional
         Overwrite existing `outfpath` (the default is False).
     parallel : bool, optional
-        Will parallelize hmmscan using GNU parallel (the default is True).
+        Will use multithreaded parallelization offered by hmmscan (the default is True).
+    gnu_parallel : bool, optional
+        Will parallelize hmmscan using GNU parallel (the default is False).
+        Note: Only one of `parallel` and `gnu_parallel` may be provided as True
     log : str, optional
         </path/to/parallel.log> (the default is None). If provided will write
         parallel log to `log`.
```

Comment:
I don't think this is been used by the function anymore and thus should be removed
```suggestion
```